form/icon-button.ftd

```diff
@@ -262,15 +262,30 @@ button.button-variant variant: solid
 button.button-width width: default
 button.button-size size: medium
 boolean outline: false
-button.button-kind kind: primary
+button.button-kind kind:
```

Comment:
Keep default. You want the user to give as minimum ui-related data as possible, yet it should look just fine.